src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let result = {};
```

Comment:
3.[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)